how-to-use-azureml/automated-machine-learning/sparse-data-train-test-split/auto-ml-sparse-data-train-test-split.yml

```diff
@@ -0,0 +1,8 @@
+name: auto-ml-sparse-data-train-test-split
```

Comment:
This file is not in AzureMlCli and should not be pushed to GitHub.
It is generated as part of running the associated notebook.